build_projects/dotnet-cli-build/CrossgenDirectory.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Compression;
+using System.Runtime.InteropServices;
+using System.Net.Http;
+using Microsoft.Build.Utilities;
+using Microsoft.DotNet.Cli.Build.Framework;
+using Microsoft.DotNet.InternalAbstractions;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
```

Comment:
This is repeated for all these new custom Task files.

---
In reply to: [69322766](https://github.com/dotnet/cli/pull/3737#discussion_r69322766) [](ancestors = 69322766)